tools/echoserver.go

```diff
@@ -89,6 +92,7 @@ func RequestHandler(response http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	// Go libs have no gzip support for HTTP responses, sending it uncompressed.
```

Comment:
Comment does not seem to match the code? (No guarantee that intermediary proxy will not compress?)